qiskit/tools/__init__.py

```diff
@@ -18,15 +18,19 @@
 .. currentmodule:: qiskit.tools
 
 Parallel Routines
-=================
+-----------------
+
+A helper function for calling a custom function with python map
 
 .. autosummary::
    :toctree: ../stubs/
 
    parallel_map
 
 Monitoring
-==========
+----------
+
+A helper module to get IBM backend information and submitted job status
```

Comment:
```suggestion
A helper module to get IBM backend information and submitted job status.
```

Also, I don't see any examples for `backend_monitor` and `backend_overview`, I think those should be added